src/main/scala/com/gu/fulfilmentLookup/Config.scala

```diff
@@ -4,8 +4,20 @@ import java.lang.System.getenv
 
 trait Config {
   def stage: String
+  def salesforceUrl: String
+  def salesforceClientId: String
+  def salesforceClientSecret: String
+  def salesforceUsername: String
+  def salesforcePassword: String
+  def salesforceToken: String
 }
 
 object EnvConfig extends Config {
```

Comment:
Raising this PR has reminded me that I'm not super keen on storing private credentials in env variables due to the need to encrypt/decrypt them. I'm considering swapping to using a file from S3 (will do this in a separate PR though)

Reply:
fair enough, I was more thinking from a quanitity point of view - if you want to delete and cloudform again it's quicker if you just choose CODE or PROD and it finds the file.